clar2wasm/src/wasm_generator.rs

```
@@ -304,9 +306,15 @@ impl Deref for BorrowedLocal {
 
 impl WasmGenerator {
     pub fn new(contract_analysis: ContractAnalysis) -> Result<WasmGenerator, GeneratorError> {
-        let standard_lib_wasm: &[u8] = include_bytes!("standard/standard.wasm");
+        let wasm_path = std::env::var("STANDARD_WASM_PATH").unwrap_or_else(|_| {
+            format!("{}/src/standard/standard.wasm", env!("CARGO_MANIFEST_DIR"))
```

Do we still need this if the env variable is set in the CI?

---

I should let it throw and error when the variable is not set